src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -8,8 +8,8 @@
       "OpenCover": "4.6.166",
       "ReportGenerator": "2.3.1",
 
-      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0023",
-      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0023",
+      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0025",
```

Comment:
Yes, the CoreFX PR with the buildtools update will update those also. 

>allowing folks to override your PackageVersion properties in the targets file

What would this look like? Would it be as simple as adding a property for the version (e.g. 1.0.0-alpha-build0023) instead of hard coding it? The relationship between the project.json files for buildtools/test-runtime, corefx/common/test-runtime, and the project itself is still a bit blurry to me.